README.md

```diff
@@ -71,7 +71,7 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+Helper method because they are used for logic that affects how things are displayed in the view.  
```

Comment:
:+1:

For a tech interview, I might also say something about why it isn't the role of the model or controller.